### ISSUE TITLE
Datagrid show/hide column selector on voice over reads text 'All columns selected', but not displayed on the UI

### ISSUE THREAD

**Issue by pratikgala** [vmware/clarity#4782]
## Describe the accessibility issue

Datagrid Show/Hide column selector on voice over reads text 'All columns selected' but the UI does not show the same text displayed. Apps using datagrid have the same issue.

## WCAG Guideline

[Issue WCAG20] - Inform assistive technology of changes in content.
(Show Hide Columns Dialogs) There is text that reads "All Columns selected", but this text is not updated when the user toggles any of the check boxes.

## How to reproduce

1.  Go to 'Clarity documentation for datagrid'
2.  Click on Show/Hide columns https://clarity.design/documentation/datagrid/hide-show
3.  Scroll down to the datagrid table, enable voice over, click on show/hide table.
4.  See error on voice over as well as by inspecting the popup box.

Text: All column selected in voice over:

![Screenshot 2020-07-13 at 17 08 20](https://user-images.githubusercontent.com/3613248/87385846-f9ac5680-c553-11ea-8f6c-2a01869cf1d4.png)

Inspecting the popup box in chrome, Dom contains the text, but not displayed in the UI:

![Screenshot 2020-07-13 at 17 09 16](https://user-images.githubusercontent.com/3613248/87385849-ff09a100-c553-11ea-8863-103a365cd9f0.png)

## Expected behavior

Ensure that this message is updated or that it is removed from the DOM altogether since the information is already communicated by the checkbox states.

## Versions

**App**

- Angular: [8.0.0]
- Node: [10]
- Clarity: [2.3.8]

**Device:**

- Type: [MacBook]
- OS: [iOS]
- Browser [chrome]
- Version [83]
- Screen Reader: [VoiceOver]

For screen readers, we primarily support the following combinations, and other combinations will be considered on a case by case basis.

- VoiceOver + Safari (Apple)

**Comment by sumegha26** [comment]
I have observed this,  ``All Columns Selected`` text is ``clr-sr-only`` meaning clarity readonly text. These texts are provided to only screen reader and not focusable also not displayed on UI. This should be as designed.

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
